debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9ConstantPoolCommand.java

```diff
@@ -0,0 +1,507 @@
+import com.ibm.j9ddr.vm29.pointer.generated.J9MethodPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ObjectPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMClassRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMConstantDynamicRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMConstantPoolItemPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMInterfaceMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodHandleRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodTypeRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMStaticFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMStringRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMClassPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMClassRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMConstantDynamicRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMConstantPoolItemPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodHandleRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodTypeRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMNameAndSignaturePointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMSingleSlotConstantRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMStringRefPointer;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ClassHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ObjectHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9RASHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ROMClassHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9UTF8Helper;
+import com.ibm.j9ddr.vm29.structure.J9ConstantPool;
+import com.ibm.j9ddr.vm29.structure.J9Constants;
+import com.ibm.j9ddr.vm29.structure.J9VmconstantpoolConstants;
+import com.ibm.j9ddr.vm29.structure.J9Consts;
+import com.ibm.j9ddr.vm29.structure.J9NonbuilderConstants;
+import com.ibm.j9ddr.vm29.types.UDATA;
+
+public class J9ConstantPoolCommand extends Command {
+	private static final HashMap<Long, String> cpTypeToString = new HashMap<>();
+	private static final int J9_METHOD_INDEX_SHIFT = 8;
+	private static final int J9_ITABLE_INDEX_SHIFT = 10;
+	private static final int ARG_COUNT_MASK = ~0xFF;
+
+	static {
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_CLASS, "Class");
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_STRING, "String");
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_INT, "Int");
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_FLOAT, "Float");
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_LONG, "Long");
+		cpTypeToString.put(J9ConstantPool.J9CPTYPE_DOUBLE, "Double");
```

Comment:
These should be "int", "float", "long" and "double" (with a lowercase first character).